src/game/settings.c

```diff
@@ -40,9 +40,9 @@ static struct {
 static void load_default_settings(void)
 {
     data.fullscreen = 1;
-    data.window_width = 800;
-    data.window_height = 600;
-    
+    data.window_width = 960;
```

Comment:
I don't like changing the default resolution for everyone. Can we make this vita-specific?

Reply:
Good catch! It was a temporary thing while porting Julius and I forgot to remove it.